doc/tutorials/sandbox_config_ui.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.            sb_config.save('EXTERNAL_HOST_NAME', inputs['db name'].value)
</pre><p>I think, for the keys in the configuration would be an enum useful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>